auth-service/src/test/kotlin/uk/gov/communities/delta/controllers/DeltaUserRegistrationControllerTest.kt

```diff
@@ -32,6 +32,14 @@ class DeltaUserRegistrationControllerTest {
         }
     }
 
+    @Test
+    fun testRegisterPageFromSSO() = testSuspend {
+        testClient.get("/register?sso_email=user@example.com").apply {
```

Comment:
Should this be encoded? Or is that not necessary?

Reply:
It does work, but it probably should be